windows/tools/nocli/NearObjectCliHandlerWindows.cxx

```diff
@@ -26,6 +26,11 @@ using namespace windows::devices;
 using namespace winrt::Windows::Devices;
 using namespace winrt::Windows::Foundation;
 
+namespace winrt::nof
```

Comment:
Looks good, could you run the auto-formatter on this file?

Reply:
Yep, seems I didn't save the results for this.